### ISSUE TITLE
Start mining on not MAIN/LAUNCHER activity

### ISSUE THREAD

**Issue by Letme** [theapache64/coin_hive_android_sdk#7]
As I figured out via #4 you need to extend your MAIN or LAUNCHER activity with `BaseCoinHiveActivity` for mining to start. If you extend any other it does not have effect.

Feature request would be to start mining on any other activity as well or via the startMining() command.

**Comment by theapache64** [comment]
As of now, if you want to use an activity as a miner, you should extend the activity from `BaseCoinHiveActivity`. If you don't want to mine using that activity, extend it from other activity class like 'Activity' or 'AppCompatActivity'.. etc.. You get my point ?

**Comment by Letme** [comment]
Yes, but your point does not work unless the activity that is run is MAIN or LAUNCHER (in Manifest). I have made a test. I have `StartActivity`, which calls `SecondActivity`. `StartActivity` is marked as MAIN and LAUNCHER, while `SecondActivity` is just DEFAULT.

Case 1 (mining starts):
`StartActivity extends BaseCoinHiveActivity` , calls `SecondActivity extends BaseCoinHiveActivity` , miner is running.

Case 2 (this report):
`StartActivity extends Activitiy` and calls `SecondActivity extends BaseCoinHiveActivity`, mining is not working. This is the only change I made in both activities and I always confirm that `Application` activity initalizes CoinHive object.
